frontend/src/components/account/EditAccountPage.js

```diff
@@ -469,6 +505,51 @@ export default function EditAccountPage({
             />
           </div>
         );
+      } else if (i.type === "selectwithtext") {
+        const valuesThatDisableField = [1, 3]; // we can manually set which types we don't want to have this feature for
+        const truthValues = valuesThatDisableField.map((val) => editedAccount.types.includes(val));
+        const shouldTextFieldBeDisabled =
+          truthValues.includes(true) || editedAccount.types.length === 0;
```

Comment:
This is a quite overcomplicated check but with the new setup of storing this info directly in the OrganizationTags model this will be much simpler anyways

Reply:
Removed & refactored to use backend info.